tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -33,12 +33,14 @@ const {
   ensureApplicationMapping,
   patchApplicationGateway,
   eventingSubscription,
-  k8sDelete
+  k8sDelete,
+  getSecretData
 } = require("../../../utils");
 
 const {
   registerOrReturnApplication,
 } = require("../../../compass");
+const { OAuthToken, OAuthCredentials } = require("../../../lib/oauth");
```

Comment:
Just to be consistent with the rest of the file

```suggestion
const { 
  OAuthToken, 
  OAuthCredentials
} = require("../../../lib/oauth");
```